src/routes/Onboarding/DesktopOnboardingRoutes.tsx

```diff
@@ -0,0 +1,634 @@
+        />
+      </OnboardingCard>
+    </div>
+  )
+}
+
+function CodePane() {
+  // Highlight the feature tree pane button if it's present
+  useOnboardingHighlight('code-pane-button')
+
+  // Open the code pane on mount, close on unmount
+  useOnboardingPanes(['code'])
+
+  return (
+    <div className="fixed inset-0 z-50 p-8 grid justify-center items-end">
+      <OnboardingCard className="col-start-3 col-span-2">
+        <h1 className="text-xl font-bold">KCL Code</h1>
+        <p className="my-4">
+          This is the KCL Pane. KCL (KittyCAD Language) is a scripting language
+          we created to describe CAD geometries. This code is the source of
+          truth, everything you do to the model will change the code.
+        </p>
+        <p className="my-4">
+          KCL boasts other scripting features such as imports, functions and
+          logic. Not only can you edit your geometry from the feature tree, but
+          you can also edit the code directly.
+        </p>
+        <OnboardingButtons
+          currentSlug="/desktop/code-pane"
+          platform="desktop"
+        />
+      </OnboardingCard>
+    </div>
+  )
+}
+
+function ProjectPane() {
+  const thisOnboardingStatus: DesktopOnboardingPath = '/desktop/project-pane'
+  // Highlight the feature tree pane button if it's present
+  useOnboardingHighlight('files-pane-button')
+
+  // Open the code pane on mount, close on unmount
+  useOnboardingPanes(['files'])
+
+  return (
+    <div className="fixed inset-0 z-50 p-8 grid justify-center items-end">
+      <OnboardingCard className="col-start-3 col-span-2">
+        <h1 className="text-xl font-bold">Files Pane</h1>
+        <p className="my-4">
+          The next pane is the Project Files Pane. Here you can see all of the
+          files you have in this project. This can be other KCL files as well as
+          external CAD files (STEP, STL, OBJ, etc.).
+        </p>
+        <OnboardingButtons
+          currentSlug={thisOnboardingStatus}
+          platform="desktop"
+        />
+      </OnboardingCard>
+    </div>
+  )
+}
+
+function OtherPanes() {
+  const thisOnboardingStatus: DesktopOnboardingPath = '/desktop/other-panes'
+  // Highlight the log and variable panes button if it's present
+  useOnboardingHighlight('logs-pane-button')
+  useOnboardingHighlight('variables-pane-button')
+
+  // Open the panes on mount, close on unmount
+  useOnboardingPanes(['logs', 'variables'])
+
+  return (
+    <div className="fixed inset-0 z-50 p-8 grid justify-center items-end">
+      <OnboardingCard className="col-start-3 col-span-2">
+        <h1 className="text-xl font-bold">Other panes</h1>
+        <p className="my-4">
+          These last two panes are the Variables Pane and Logs Pane. The
+          Variables pane will display the numeric values of any parameters you
+          made, along with other entities and types created in your in your
+          file. The Logs pane will show error logs.
```

Comment:
Capitalize for consistency?

```suggestion
          file. The Logs Pane will show error logs.
```

Reply:
I agree with these but we can do them in a follow-up